### DISCUSSION TITLE
🎉 Hyperswitch Joins Hacktoberfest 2024! 🌍💻

### DISCUSSION THREAD

**Discussion by gorakhnathy7** [juspay/hyperswitch#6332] [Announcements]
We’re **excited** to announce that **Hyperswitch** is part of **Hacktoberfest 2024**! Whether you’re a coding pro or just starting out, this is your chance to contribute to our open-source payment platform and earn some awesome rewards! 🎁

## How to Get Started

1. **Sign Up for Hacktoberfest**:  
   Register on the Hacktoberfest site to join in! ✅

2. **Explore the Hyperswitch Repo**:  
   Explore our [GitHub repo](https://github.com/juspay/hyperswitch) and **Star ⭐** it. Look for issues tagged `hacktoberfest`, `good first issue`, or `help wanted`.

3. **Understand the Project**:  
   Read our **README** and **Contribution Guidelines** 📖 before you start. For **paid contributions**, check out our **Terms & Conditions**.

4. **Submit Your Pull Requests**:  
   Follow the contribution guidelines and submit your PRs between **October 1st and October 31st**.

## Contribution Rules 👇

- **One issue at a time**: Comment on the issue you want to claim, and we’ll assign it to you. Once your PR is merged or closed, you can take on another.
- Follow coding standards and run tests before submitting. 🧹
- Use clear commit messages and branch names like `fix/issue-<issue-number>`.
- Each PR should address **one issue only**. 🎯

## Review Process 🕵️‍♀️

Once your PR is submitted, a maintainer will review it. Be prepared for feedback and edits ✏️. We’ll do our best to review quickly!

## Testing 🧪

Ensure your code passes all tests. If you’re adding new functionality, consider writing tests to ensure smooth performance! 🚦

## Rewards 🎁

- **1 Issue Completed**: Earn a Hyperswitch t-shirt 👕!  
- **3+ Issues Completed**: Unlock an exclusive Hyperswitch Swag Kit 🎉!

## No-Code Contributions 🖋️

Non-coders can contribute by improving documentation or writing blogs. These contributions also count toward Hacktoberfest!

## Need Help? 🤔

Reach out on **[Slack](https://join.slack.com/t/hyperswitch-io/shared_invite/zt-362xmn7hg-ujdw8Wvx_~BgNTLrCcdCPw)** if you have questions. We’re here to help! 💬

---

Let’s make **[Hacktoberfest 2024](https://hyperswitch.io/hacktoberfest)** amazing! 🎇 We look forward to your contributions. Happy hacking! 💻✨

---

**Comment by ApiZimbah** [comment]
Me🩷 hakken
